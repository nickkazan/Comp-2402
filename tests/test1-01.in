Black
Yellow
Green
Red
Pink
Orange
Blue
Purple
