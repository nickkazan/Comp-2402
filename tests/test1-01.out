Purple
Blue
Orange
Pink
Red
Green
Yellow
Black
